.github/workflows/stale.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark stale issues
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
```

Comment:
I see there's a [v2](https://github.com/actions/stale/tree/v2) and [v3](https://github.com/actions/stale/tree/v3) of this action -- any reason for choosing v1? I think v3 might be pretty cool because it removes the stale label on issues when there is recent activity: https://github.com/actions/stale/releases/tag/v3.0.0.

Reply:
Good catch! Fixed.